### PR TITLE
Add episode 6

### PR DESCRIPTION
**Pull request by McInkay**

### CODE REVIEW

**Comment by McInkay** [comment]
Still need to do the timings and the size/time for the file once it is finalised.

I'm not sure what sections would be best in this episode. Any thoughts, @smillie ?

**Comment by McInkay** [comment]
This should be good to go this evening now.

File: content/episode/006-dev-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+date: 2019-02-13T21:00:00+00:00
+
+author: "Al McKinlay and Andrew Smillie"
+aliases: []
+categories: []
+---
+
+## Timings
+
+[00:25] Introduction to Remote Working
```
Comment (smillie) [inline content/episode/006-dev-onboarding.md]: Developer onboarding

File: content/episode/006-dev-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Developer Onboarding"
+episode: "6"
+Description: "Al and Andrew talk about developer onboarding. How do you onboard new developers and ?"
```
Comment (smillie) [inline content/episode/006-dev-onboarding.md]: Maybe drop the ‘and ?’

Comment (McInkay) [reply content/episode/006-dev-onboarding.md]: 🤦‍♂️

File: content/episode/006-dev-onboarding.md (hunk 1 above)
Comment (McInkay) [reply content/episode/006-dev-onboarding.md]: 🤦‍♂️

File: content/episode/006-dev-onboarding.md (hunk 2 above)
Comment (McInkay) [reply content/episode/006-dev-onboarding.md]: Or I could finish the sentence. Done.

File: content/episode/006-dev-onboarding.md (hunk 1 above)
Comment (McInkay) [reply content/episode/006-dev-onboarding.md]: This is why we do reviews. Done.

**Pull request merged**